lineage/Analyze.py

```diff
@@ -173,3 +137,45 @@ def run_Results_over(output: list, parallel=True) -> list:
 
     prom_holder = [exe.submit(Results, *x) for x in output]
     return [prom.result() for prom in prom_holder]
+
+def permute_states(tHMMobj: Any, pred_states: list) -> Tuple[Any, list]:
```

Comment:
So if I'm following, the true states are stored in `tHMMobj.X.output_lineage`. If the tHMM object is fit, and has the true values stored in it, why do we need `pred_states`? Can't the predicted states be generated by calling Viterbi using the data (which is stored inside the tHMM object)? It'd be nice to get rid of a redundant argument if possible.